tn_api_v2_core.yml

```diff
@@ -0,0 +1,948 @@
+openapi: "3.0.0"
+
+info:
+    version: 2.0.0
+    title: Trip Ninja API Documentation
```

Comment:
Should we change the title to reflect this being the newer core non-emulation architecture, or is this the simply Trip Ninja 2.0. wdyt?

Reply:
i think its just 2.0, its ok not to change it